engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -1400,16 +1400,20 @@ namespace dmGameSystem
             // Since the animation referred to the old texture, we need to update it
             if (res == dmGameObject::PROPERTY_RESULT_OK)
             {
-                uint32_t* anim_id =  GetTextureSet(component, component->m_Resource)->m_AnimationIds.Get(component->m_CurrentAnimation);
+                TextureSetResource* texture_set = GetTextureSet(component, component->m_Resource);
+                uint32_t* anim_id =  texture_set->m_AnimationIds.Get(component->m_CurrentAnimation);
                 if (anim_id)
                 {
                     PlayAnimation(component, component->m_CurrentAnimation, GetCursor(component), component->m_PlaybackRate);
                 }
                 else
                 {
-                    component->m_Playing = 0;
```

Comment:
Why are we no longer setting m_Playing to 0?

Reply:
because it is typo ( good catch!

Reply:
Fixed